code/game/objects/items/hot_potato.dm

```diff
@@ -52,6 +52,9 @@
 	var/atom/location = loc
 	location.visible_message(span_userdanger("[src] [detonate_explosion? "explodes" : "activates"]!"), span_userdanger("[src] activates! You've ran out of time!"))
 	if(detonate_explosion)
+		var/mob/living/m = loc
+		if(m.get_active_held_item()==src)
```

Comment:
what happens if it isn't our active held item but otherwise in our inventory? (is that forbidden for some reason or) similarly, what happens if we blow up on something that isn't a mob. is that possible?

Reply:
i dont think the hot potato can be attatched to anything thats not a mobs, and the potato cannot be placed in your inventory

Reply:
vibin